rdr_service/dao/participant_summary_dao.py

```diff
@@ -1118,7 +1119,14 @@ def update_ehr_status(summary, update_time):
 
     @classmethod
     def get_all_consented_participant_ids(cls, session):
-        db_results = session.query(ParticipantSummary.participantId).all()
+        db_results = session.query(
```

Comment:
Would there be value in excluding ghost ids, or do they have to be treated as potentially valid?

Reply:
If they're marked as ghosts and we're getting survey responses from them, then there's likely some other investigation that needs to be happening and filtering them out here would cause us to miss that.